AdvanceTaskNunit/AdvanceTaskNunit/Tests/LangaugeTest.cs

```diff
@@ -0,0 +1,73 @@
+﻿using AdvanceTaskNunit.Components;
+using AdvanceTaskNunit.Pages;
+using AdvanceTaskNunit.Steps;
+using AdvanceTaskNunit.Test_Model;
+using AdvanceTaskNunit.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AdvanceTaskNunit.Tests
+{
+    [TestFixture]
+    public class LangaugeTest : CommonDriver
+    {
+        ProfileTabComponent profileTabComponentObj;
+        LanguageStep languageStepObj;
+        CleanUp cleanUpObj;
+
+
+
+#pragma warning disable
+
+        public LangaugeTest()
+        {
+          
+            profileTabComponentObj = new ProfileTabComponent();
+            languageStepObj = new LanguageStep();
+            cleanUpObj = new CleanUp();
+
+        }
+
+        [Test, Order(1),Description("Verify that a language and its level can be added")]
+
+        public void TestAddLanguage()
+        {
+            
+            cleanUpObj.ClearExistingdata();
```

Comment:
this not correct, note you need to clean call data before any test runs and then clean data that the test generates at the end of each test! (do not clean all the data but only the one that the test is generating).